### PR TITLE
Drag and drop data upload (work in progress)

### PR DESCRIPTION
**Pull request by LiamBurke24**
Changed filetype to .php and added comments to begin discussion about how to adapt the original util to our needs.

## Description

added basic comments and changed filetype 

## Motivation and Context

Web files should be .php (not .txt as before). 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue) 
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the CHANGELOG.md.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by LiamBurke24** [comment]
I am unsure where I need to insert the server/ folder in the drag and drop script. Googling functions hasn't been helpful.

**Comment by mdietze** [comment]
This isn't a language I know well, but it looks like you'd want to put code to save the file near the end of the read.file function after confirming that the upload is complete.

**Comment by LiamBurke24** [comment]
I think this should work. Do you need to merge this for me to be able to experiment with it?

File: web/drag.and.drop.upload.php [hunk 1]
```diff
@@ -81,6 +81,8 @@
       formdata: document.getElementById('formdata'),
       progress: document.getElementById('progress')
     },
+    
+    <!-- This could be problematic: What generic mimetypes should we include? -->
```
Comment (mdietze) [inline web/drag.and.drop.upload.php]: I wouldn't restrict by mimetype at all

File: web/drag.and.drop.upload.php [hunk 2]
```diff
@@ -129,7 +131,8 @@ function readfiles(files) {
     // now post a new XHR request
     if (tests.formdata) {
       var xhr = new XMLHttpRequest();
-      xhr.open('POST', '/devnull.php');
+      // @robkooper says xhr.open fetches the file from the server side 
+      xhr.open('POST', '/dbfiles'); // changed destination directory
```
Comment (mdietze) [inline web/drag.and.drop.upload.php]: Where exactly is /dbfiles relative to in your file structure -- does that imply a subfolder in the web folder or an absolute path relative to the root directory?

File: web/drag.and.drop.upload.php (hunk 1 above)
Comment (LiamBurke24) [inline web/drag.and.drop.upload.php]: oh, haha. Yup. That would be better!

**Comment by mdietze** [comment]
The issues with this PR haven't been addressed, and the exact same code seems to be in #1578. @LiamBurke24 should we close this?

**Comment by tonygardella** [comment]
Liam said I can close this.

**Pull request closed without merging**